### PR TITLE
Please add PRISMA App to the Big-Map registry

### PR DESCRIPTION
**Pull request by eibfl-dtu**

### CODE REVIEW

File: apps.yaml [hunk 1]
```diff
@@ -97,3 +97,15 @@ HELAO:
...
+  metadata:
+    title: PRISMA
+    description: |
+        An app for for high-throughput analysis of spectra. More details in the app homepage and peer-reviewed article:
+    authors: Eibar Flores
+    external_url: https://github.com/BIG-MAP/PRISMA
+    logo: https://raw.githubusercontent.com/BIG-MAP/PRISMA/main/docs/figures/logo.png
+    state: development
+  categories:
+     - data analysis
```
Comment (csadorf) [inline apps.yaml:110]: This category does currently not exist. You can add it to the [`categories.yml`](https://github.com/BIG-MAP/big-map-registry/blob/main/categories.yaml) if you like.

File: apps.yaml [hunk 2]
```diff
@@ -97,3 +97,15 @@ HELAO:
   categories:
     - technology-lab
     - technology-active-learning
+prisma:
+  metadata:
+    title: PRISMA
+    description: |
+        An app for for high-throughput analysis of spectra. More details in the app homepage and peer-reviewed article:
```
Comment (csadorf) [inline apps.yaml:104]: Do you want to add a link to the article or is the link provided under `external_url` sufficient?

**Review by csadorf — CHANGES_REQUESTED** [review]

File: apps.yaml (hunk 2 above)
Comment (eibfl-dtu) [reply apps.yaml:104]: The external link is sufficient. The scientific paper is accepted but it does not have DOI yet. I will add the DOI directly in the site under external_url

File: apps.yaml (hunk 1 above)
Comment (eibfl-dtu) [reply apps.yaml]: I have now added the category to categories.yml

**Review by csadorf — CHANGES_REQUESTED** [review]
Ok, I think there is one more little typo in there, but otherwise it looks good.

File: apps.yaml (hunk 2 above)
Comment (eibfl-dtu) [reply apps.yaml]: Thanks! Now corrected, also the 'An app for for' i saw there.

**Review by csadorf — APPROVED** [review]

**Pull request merged**